### DISCUSSION TITLE
How do I get these Tabs?

### DISCUSSION THREAD

**Discussion by DevNugget** [neovide/neovide#911] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/84568105/130391587-19643895-2080-40fd-bb35-29d17b40e46d.png)

**Comment by Kethku** [comment, marked as answer]
Those tabs are provided by vim-airline
https://github.com/Kethku/vim-config/blob/main/fnl/module/plugins.fnl#L8-L9
And are configured here: https://github.com/Kethku/vim-config/blob/main/fnl/module/settings.fnl#L12-L13
And here: https://github.com/Kethku/vim-config/blob/main/fnl/module/settings.fnl#L17-L23

**Reply by Kethku** [reply]
Note: this is written using fennel lang which compiles down to lua. Similar config is possible in vimscript directly
